src/main/java/com/endava/internship/collections/StudentList.java

```diff
@@ -1,145 +1,459 @@
 package com.endava.internship.collections;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.List;
-import java.util.ListIterator;
+import java.util.*;
 
 public class StudentList implements List<Student> {
+    private final static int DEFAULT_CAPACITY = 10;
+    private int size = 0;
+    private int capacity = DEFAULT_CAPACITY;
+    private Student[] students;
+
+    public StudentList() {
+        students = new Student[DEFAULT_CAPACITY];
+    }
+
+    public StudentList(int capacity) {
+        if(capacity<0)
```

Comment:
it's not wrong, but usual for having a better readability of the code, is practiced using of brackets "{}"